### ISSUE TITLE
Failed to resolve vue/compiler-sfc.

### ISSUE THREAD

**Issue by floriancargoet** [laynezh/vite-plugin-lib-assets#28]
Since the last version (0.5.4), my build fails with:
```
[vite-plugin-lib-assets] Failed to resolve vue/compiler-sfc.
vite-plugin-vue-setup-name requires vue (>=2.7.0) to be present in the dependency tree.
error during build:
Error: Failed to resolve vue/compiler-sfc.
vite-plugin-vue-setup-name requires vue (>=2.7.0) to be present in the dependency tree.
```

It worked with 0.5.3.
I'm not using vue at all.

**Comment by coder-layne** [comment]
Thank you for your feedback, the introduction of the Vue SFC parser should be delayed util  it is actually used, I'll fix it later.

**Comment by coder-layne** [comment]
I just released v0.5.5, give it a try to see if it solves the issue.

**Comment by floriancargoet** [comment]
It seems to be working, thank you!
